src/System.Text.Encoding.CodePages/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -0,0 +1,5 @@
+Compat issues with assembly System.Text.Encoding.CodePages:
+CannotRemoveBaseTypeOrInterface : Type 'System.Text.CodePagesEncodingProvider' does not inherit from base type 'System.Text.EncodingProvider' in the implementation but it does in the contract.
```

Comment:
@tarekgh should these APIs be exposed in the reference assembly?

Reply:
I believe the problem here is in the reference assembly should have CodePagesEncodingProvider inherit from EncodingProvider as we do in the implementation. I don't know why we didn't do that in the first place when we have created the ref assembly. 

https://github.com/tarekgh/corefx/commit/40723b795917f2e98b64d45c17effb2546dd0cae

CC @ericstj 

I think we should fix the ref assembly. please let me know if you want me to do it.

Reply:
@tarekgh thanks for confirming I will simply update the ref with this PR.